en/flight_controller/cuav_v5_nano.md

```diff
@@ -134,14 +135,18 @@ We recommend that when using USB with the *V5 nano* you *also connect the power
 - *Fixed:* -
 
 
-#### Using JTAG for hardware debugging
+#### Using JTAG for hardware debugging {#issue_jtag}
 
 `DSU7` FMU Debug Pin 1 is 5 volts - not the 3.3 volts of the CPU.
 
 Some JTAG use this voltage to set the IO levels when communicating to the target.
 
 For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts of `DSM`/`SBUS`/`RSSI` pin 4 as Pin 1 on the debug connector (`Vtref`).
 
+#### HV\_PM power module output is unfused {#issue_pm_unfused}
```

Comment:
@davids5 This is new issue crated for power module output. Please check.

Note, duplicated in the other doc too.

Reply:
Please make it Clear Good - 1) that power needs to be off while connecting peripherals . 

Add: Improper wiring can lead to equipment damage or personal harm.

Reply:
Done. See https://github.com/PX4/px4_user_guide/pull/538#discussion_r307573962